lib/sales_analyst.rb

```diff
@@ -0,0 +1,24 @@
+class SalesAnalyst
+
+  attr_reader :items,
+              :merchants
+
+  def initialize(items, merchants)
+    @merchants = merchants
+    @items = items
+  end
+
+  def merch_item_creation(items, merchants)
+    return_hash = {}
+    merchants.all.each do |merchant|
+    items.all.each do |item|
```

Comment:
Beautiful setup; hound might yell at you about the lack of indentation on the items.all, though